### ISSUE TITLE
Convert data from depth map to point cloud

### ISSUE THREAD

**Issue by David-Estevez** [roboticslab-uc3m/textiles#35]
Convert data from depth map to point cloud

**Comment by jgvictores** [comment]
In teo-head ( [header](https://github.com/roboticslab-uc3m/teo-head/blob/e0b14e5e8a596daec225de2b5e89ceca3e8e3808/programs/colorRegionDetection/SegmentorThread.hpp#L28-L35) and [implementation](https://github.com/roboticslab-uc3m/teo-head/blob/e0b14e5e8a596daec225de2b5e89ceca3e8e3808/programs/colorRegionDetection/SegmentorThread.cpp#L236-L241) ) we use a linear transformation from pixels to mm:
```c
#define DEFAULT_FX_D          525.0  // 640x480
#define DEFAULT_FY_D          525.0  //
#define DEFAULT_CX_D          319.5  //
#define DEFAULT_CY_D          239.5  //
#define DEFAULT_FX_RGB        525.0  //
#define DEFAULT_FY_RGB        525.0  //
#define DEFAULT_CX_RGB        319.5  //
#define DEFAULT_CY_RGB 239.5 //
```

**Comment by David-Estevez** [comment]
Resolution of a single frame wasn't enough, nevertheless the code is merged in 298daae as it might be useful in the future.
